.github/README.md

```diff
@@ -98,7 +98,7 @@ Working on the 🐘 big Llama 3.1 405B model:
 
 ## Model Fine Tuning:
 
-It is often not enough to run inference on the model. 
+It is often not enough to run inferences on the model.
```

Comment:
Let's revert back to inference as I do not seem to think it is a grammatical error.